pinot-core/src/main/java/org/apache/pinot/core/common/MinionConstants.java

```diff
@@ -92,6 +92,8 @@ private MinionConstants() {
   // Generate segment and push to controller based on batch ingestion configs
   public static class SegmentGenerationAndPushTask {
     public static final String TASK_TYPE = "SegmentGenerationAndPushTask";
+    public static final String CONFIG_NUMBER_CONCURRENT_TASKS_PER_INSTANCE =
+        "pinot.minion.task.generator.SegmentGenerationAndPushTask.numConcurrentTasksPerInstance";
```

Comment:
Simplify this key to `SegmentGenerationAndPushTask.numConcurrentTasksPerInstance`? The current one looks like an instance config key